kafka_consumer/tests/conftest.py

```diff
@@ -53,10 +54,14 @@ def dd_environment(mock_local_kafka_hosts_dns, e2e_instance):
             # Advertising the hostname doesn't work on docker:dind so we manually
             # resolve the IP address. This seems to also work outside docker:dind
             # so we got that goin for us.
-            'KAFKA_HOST': HOST_IP
+            'KAFKA_HOST': HOST_IP,
+            'BOOTSTRAP_SERVER_FLAG': _get_bootstrap_server_flag(),
         },
     ):
-        yield e2e_instance, E2E_METADATA
+        yield {
+            'instances': [e2e_instance],
+            'init_config': {'zk_timeout': 30, 'kafka_timeout': 30},
```

Comment:
The timeouts here fix the connection timeouts, NoBrokersAvailable and UnrecognizedBrokerVersion errors